bevy_lint/src/lint.rs

```diff
@@ -163,25 +156,9 @@ macro_rules! declare_bevy_lint_pass {
     (
         $(#[$attr:meta])*
         $vis:vis $name:ident => [$($lint:expr),* $(,)?],
-
-        $(
-            @default = {
-                $($default_field:ident: $default_ty:ty = $default_value:expr),* $(,)?
-            },
-        )?
     ) => {
         $(#[$attr])*
-        $vis struct $name {
-            $($($default_field: $default_ty),*)?
-        }
-
-        impl ::std::default::Default for $name {
-            fn default() -> Self {
-                Self {
-                    $($($default_field: $default_value),*)?
-                }
-            }
-        }
```

Comment:
We no longer implement `Default` for lint passes, as they are unit structs with no fields.